.vscode/settings.json

```diff
@@ -11,5 +11,6 @@
   },
   // Turn off tsc task auto detection since we have the necessary tasks as npm scripts
   "typescript.tsc.autoDetect": "off",
-  "typescript.preferences.quoteStyle": "single"
+  "typescript.preferences.quoteStyle": "single",
+  "dafny-vscode.trace.server": "verbose"
```

Comment:
Do we want this by default in the development experience? To me it makes sense, but might be an accidental commit.